tasks/bootstrap_local_vm/02_create_local_vm.yml

```diff
@@ -1,14 +1,18 @@
 ---
 - name: Create hosted engine local vm
   block:
+  - set_fact: host_key_checking="{{ lookup('config','HOST_KEY_CHECKING') }}"
```

Comment:
Last thing: ansible-lint is failing with:

[502] All tasks should be named
/home/stirabos/ovirt/ovirt-ansible-hosted-engine-setup/tasks/bootstrap_local_vm/02_create_local_vm.yml:4
Task/Handler: set_fact host_key_checking={{ lookup('config','HOST_KEY_CHECKING') }}

and yamllint complains due to a missing space after 'config',

Can you please fix it as well?

Reply:
Done!